backend/services/syson-services/src/main/java/org/eclipse/syson/util/DescriptionNameGenerator.java

```diff
@@ -251,6 +251,10 @@ public String getEdgeName(EClass eClass) {
      */
     @Override
     public String getEdgeName(String type) {
-        return this.getEdgeName(this.diagramPrefix, type);
+        return this.getEdgeName(this.getDiagramPrefix(), type);
+    }
+
+    protected String getDiagramPrefix() {
```

Comment:
Could you add `getDiagramPrefix` to the interface please and change the visibility of the implementation to public?